Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -3,7 +3,7 @@ $project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
 
 Describe "Get-GcbqDataset" {
 
-    It "should list datasets" {
+    It "should list datasets." {
```

Comment:
I don't think you really need to add "." at the end of this. If you look at the Pester [repo](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md), they don't have "." at the end of It statement.